tests/datasets/test__init__.py

```diff
@@ -0,0 +1,424 @@
+# Copyright 2018-2019 Faculty Science Limited
```

Comment:
We would normally call a file like this `test_init.py` (note just single underscore).